workflows/dags/author/author_create/author_create_init.py

```diff
@@ -55,40 +54,50 @@ def set_schema(**context):
         workflow_management_hook.partial_update_workflow(
             workflow_id=context["params"]["workflow_id"],
             workflow_partial_update_data={
-                "data": {**context["params"]["data"], "$schema": schema}
-            },
-            collection=AUTHORS,
+                "data": {**context["params"]["workflow"]["data"], "$schema": schema}
+            }
         )
 
     @task()
     def create_author_create_user_ticket(**context: dict) -> None:
-        endpoint = "/api/tickets/create"
-        request_data = {
-            "functional_category": "Author curation",
-            "template": "user_new_author",
-            "workflow_id": context["params"]["workflow_id"],
-            "subject": "test",  # TODO: set the subject and description
-            "description": "test",
-            "caller_email": "",  # leave empty
-        }
-        response = inspire_http_hook.call_api(
-            endpoint=endpoint, data=request_data, method="POST"
-        )
-        logger.info(f"Ticket created. Response status code: {response.status_code}")
-        logger.info(response.json())
+
+        workflow_data = context["params"]["workflow"]["data"]
+        email = workflow_data["acquisition_source"]["email"]
+        
+        response = inspire_http_hook.create_ticket(
+            AUTHOR_SUBMIT_FUNCTIONAL_CATEGORY,
+            "curator_new_author",
+            f"Your suggestion to INSPIRE: author {workflow_data.get('name').get('preferred_name')}",
+            workflow_data["acquisition_source"]["email"],
+            {
+                "email": email,
+                "obj_url":inspire_http_hook.get_backoffice_url(context["params"]["workflow_id"])
+                })
+
```

Comment:
this is also not formatted

Reply:
pre-commit was not yet activated for the branch